### ISSUE TITLE
No audio on XIAO ESP32S3 Cam

### ISSUE THREAD

**Issue by SinanAkkoyun** [spawn451/ESP32-CAM_Audio#9]
Hi! When adapting audio_server.h and the camera model to XIAO, the video works but audio just won't play (it won't load) without any errors in serial console. No matter in which browser I try.

Interestingly, once I load the `:82/audio` url, it doesn't load again in a new tab but the main URL to the default cam server always loads, indicating that the audio web server somehow crashes (same goes for `:83`), which would explain why I can't load audio, but I don't know why it crashes

I would be grateful for your help

**Comment by digitallyamar** [comment]
Even I am facing the same issue. I am unable to load :82/audio url, the web page just doesnt load. Were you able to proceed any further with this issue?

**Comment by SinanAkkoyun** [comment]
Nope! I call for help! My problem is that apparently the esp can't handle i2s mic and cam together as it seems

**Comment by spawn451** [comment]
Hello, It's been a long time since I last played with ESP. I just tried : http://192.168.10.65:82/audio
![image](https://github.com/user-attachments/assets/b515ac24-7f4c-457b-977d-0598f4edf75d)
And audio works! (I feel like this board is a bit messed up)

**Comment by piroposantosdev** [comment]
@spawn451 are you using INMP441 mic? Did you change something in this code? For me the audio is not working... i can see the player like the image you sent but no audio is playing

![D_NQ_NP_641717-MLB50669282926_072022-](https://github.com/user-attachments/assets/4d07c4e8-e267-4658-b61b-17e71f94e64f)

This is my esp32 cam... it'll work with your code?

**Comment by guillaumesene** [comment]
Hello! Same problem here.

When adapting audio_server.h and the camera model to XIAO ESP32S3 SENSE, the video works but audio just won't play without any errors in serial console.

Interestingly, once I load the :82/audio url, it doesn't load again in a new tab but the main URL to the default cam server always loads, indicating that the audio web server somehow crashes (same goes for :83), which would explain why I can't load audio, but I don't know why it crashes

I would be grateful for your help

**Comment by philippedc** [comment]
Which version of IDE and Expressif ?

Mine is working fine with AI thinker with:  IDE version 1.8.16 with Expressif Sytems (ESP32) version 2.0.3 
As I get many difficulties with INM441 settings, I understood I2S micro does not work with Expressif Sytems (ESP32) version 2.0.4 and after....

**Comment by guillaumesene** [comment]
IDE 2.3.4
ESP32 3.0.7

**Comment by mabuware** [comment]
If you have L/R connected to GND like in the provided schematic, you have to change this line in audio_server.h:

.channel_format = I2S_CHANNEL_FMT_ONLY_RIGHT

to

.channel_format = I2S_CHANNEL_FMT_ONLY_LEFT
